src/envo/envo-edit.owl

```diff
@@ -37862,6 +38067,1591 @@ AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibra
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_01001979> <https://orcid.org/0000-0002-4366-3088>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_01001979> "2024-12-10T22:02:32Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/ENVO_01001979> <https://en.wikipedia.org/wiki/Zooplankton>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_01001979> "zooplankton biodiversity"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001979> <http://purl.obolibrary.org/obo/ENVO_01001975>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001979> ObjectUnionOf(ObjectIntersectionOf(<http://purl.obolibrary.org/obo/ENVO_01001975> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000080> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PCO_0000002> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002473> <http://purl.obolibrary.org/obo/ENVO_01002018>)))) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000080> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PCO_0000001> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002473> <http://purl.obolibrary.org/obo/ENVO_01002018>)))))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_01001980> (phytoplankton biodiversity)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01001980> "Biodiversity which inheres in a population or community of phytoplankton.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_01001980> <https://orcid.org/0000-0002-4366-3088>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_01001980> "2024-12-10T22:02:32Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/ENVO_01001980> <https://en.wikipedia.org/wiki/Phytoplankton>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_01001980> "phytoplankton biodiversity"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001980> <http://purl.obolibrary.org/obo/ENVO_01001975>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001980> ObjectUnionOf(ObjectIntersectionOf(<http://purl.obolibrary.org/obo/ENVO_01001975> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000080> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PCO_0000002> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002473> <http://purl.obolibrary.org/obo/ENVO_01002017>)))) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000080> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PCO_0000001> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002473> <http://purl.obolibrary.org/obo/ENVO_01002017>)))))
```

Comment:
This says:

```
(biodiversity
 and ('quality of' some 
    ('ecological community'
                 and ('composed primarily of' some phytoplankter)))) or

('quality of' some 
    ('population of organisms'
     and ('composed primarily of' some phytoplankter)))
```

A disjunction of a G-D pattern and a single differentia.

Note this entails that phytoplankter population diameter is a subclass of phytoplankton biodiversity